src/main/kotlin/ch/komunumo/server/business/configuration/control/ConfigurationService.kt

```diff
@@ -33,13 +36,42 @@ object ConfigurationService {
 
     private fun loadConfiguration(): Properties {
         val properties = Properties()
-        val file = Paths.get(System.getProperty("user.home"), ".komunumo", configurationFilename).toFile()
+        val filePath = System.getenv(envVariableName) ?: ""
+        val file: File = configFile(filePath)
+
+
         if (file.exists()) {
+            ConfigurationService.logger.info { "Using configuration: " + file.toString() }
```

Comment:
Btw, you don't need to prefix the logger call with the class name. The `logger` variable is an instance variable.